src/components/LogInSignUp/Login.vue

```diff
@@ -100,7 +97,13 @@
           this.getAddressBook()
           this.retriesActiveCarts()
           this.getWallet()
-          if (response.data.is_store_owner === true) {
+          var sids = Object.keys(response.data.user_roles.store_map)
+          if (sids.length > 0) {
+            this.$store.commit('update_store_selection', sids[0])
+            axios.defaults.headers.common['storeId'] = sids[0]
```

Comment:
if you want to set the storeid header here instead of during the individual calls then i think its better to set the header in the actual update_store_selection mutation instead of here. Cause this doesnt account for multiple stores and only sets it at login vs app creation